docker-compose.test.watch.yaml

```diff
@@ -5,4 +5,4 @@ services:
     command: >
       sh -c "npm run test:watch"
     volumes:
-      - ./.git:/home/node/.git
+      - ./app:/home/node/app
```

Comment:
Is this correct to override, was this just a placeholder?

Reply:
Changed back to `./.git:/home/node/.git`